### DISCUSSION TITLE
Can't stack downloads into Queue and slower / crashing downloads

### DISCUSSION THREAD

**Discussion by dprui002** [aclap-dev/video-downloadhelper#871] [Help and Q&A]
### Your question

It seems that you can no longer stack more than your concurrent downloads?  Before it you hit your max concurrent downloads it would queue any download thereafter and begin immediately after the next download finishes.  The whole backend portal is gone so that sorta stinks unless I'm missing something?

Same with slow / crashing download speeds.  It seems to be throttling down, so much that the download is failing where as before even if it did get slow, it would never completely stop downloading and throw an error.  Max download speeds seem to be under 10mb, closer to 5mb?

### Addon & CoApp version and details

Addon & Coapp version

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

For the queuing issue, other people have reported that in this forum.  You could search for those reports.

For download speed, people keep reporting this & I keep saying the problem is with the web site, not VDH.  I have seen VDH downloading things at 40 million, 50 million bytes per second, & faster.  It depends on the web site, not VDH.  If you want me to get results to compare with yours, you have to give me URLs.  Make sure I don't need to log on to see whatever it is.  And don't just tell me the name of a web site.  Give me specific URLs of individual web pages on which videos are hosted.  I'm not going to scrounge around on some site I've never heard of looking for something that may or may not illustrate your problem.  Give me concrete examples of downloads that have been slow for you.  And let's hope the site or sites you choose doesn't/don't geoblock me.

**Reply by GrampaWildWilly** [reply]
And you need to post a proper problem report.  That is, fill in the section of your initial post with the version information.  Click the Wiki link that is at the top of every page in this forum.  On that page, down the right side you will see chapter titles.  Click the one for "I need help."  Read what you find & provide the missing information.  Above your initial post & to the right, you can see three dots.  Click that.  I think you'll be able to figure things out from there.
